test/Microsoft.Health.Common.UnitTests/Microsoft.Health.Common.UnitTests.csproj

```diff
@@ -30,7 +30,6 @@
 			<PrivateAssets>all</PrivateAssets>
 			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
 		</PackageReference>
-		<PackageReference Include="Newtonsoft.Json" Version="13.0.1" />

```

Comment:
Reason for removal?  This was recently added because the Microsoft Test package has a Newtonsoft dependency and this ensures 13.0.1 or higher is used.